src/Tizen.Multimedia/MediaTool/MediaFormatChannelPosition.cs

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Specifies the channel position of <see cref="MediaFormatAudioMimeType.Pcm"/> media format.
+    /// </summary>
+    /// <remarks>This type is based on SMPTE 2036-2-2008 standard.</remarks>
+    /// <seealso cref="MediaFormatAudioMimeType"/>
+    /// <since_tizen> 6 </since_tizen>
+    public enum MediaFormatAudioChannelPosition
+    {
+        /// <summary>
+        /// This is used for position-less channels.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        None = -3,
+
+        /// <summary>
+        /// Mono channel.
+        /// </summary>
+        /// <remarks>If user want to set this value, <see cref="AudioMediaFormat.Channel"/> should be 1.</remarks>
+        /// <since_tizen> 6 </since_tizen>
+        Mono = -2,
+
+        /// <summary>
+        /// Invalid position.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        Invaild = -1,
+
+        /// <summary>
+        /// A loudspeaker position located at far left and centered vertically with the middle layer.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        FrontLeft = 0,
+
+        /// <summary>
+        /// A loudspeaker position located at far right and centered vertically with the middle layer.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        FrontRight,
+
+        /// <summary>
+        /// A loudspeaker position located at the middle layer corresponding to the center of the television
```

Comment:
A loudspeaker located in the middle layer surrounding the center of the television screen as viewed from the seating area.